src/pages/messages/TasksPage.tsx

```diff
@@ -46,91 +51,114 @@ const StyledSearchModeButton = styled(LinkButton)({
+    queryFn: () => fetchTickets(rowsPerPage, (page - 1) * rowsPerPage, ticketQueryString),
     meta: { errorMessage: t('feedback.error.get_messages') },
   });
 
+  const ticketTypeBuckets = ticketsQuery.data?.aggregations?.type.buckets ?? [];
+  const doiRequestCount = ticketTypeBuckets.find((bucket) => bucket.key === 'DoiRequest')?.docCount;
+  const publishingRequestCount = ticketTypeBuckets.find((bucket) => bucket.key === 'PublishingRequest')?.docCount;
+  const generalSupportCaseCount = ticketTypeBuckets.find((bucket) => bucket.key === 'GeneralSupportCase')?.docCount;
+
+  const ticketStatusBuckets = ticketsQuery.data?.aggregations?.status.buckets ?? [];
+  const ticketNewCount = ticketStatusBuckets.find((bucket) => bucket.key === 'New')?.docCount;
+  const ticketPendingCount = ticketStatusBuckets.find((bucket) => bucket.key === 'Pending')?.docCount;
+  const ticketCompletedCount = ticketStatusBuckets.find((bucket) => bucket.key === 'Completed')?.docCount;
+  const ticketClosedCount = ticketStatusBuckets.find((bucket) => bucket.key === 'Closed')?.docCount;
+
+  // NVI data
+  const [nviStatusFilter, setNviStatusFilter] = useState<NviStatusFilter>({
+    Pending: true,
+    Approved: false,
+    Rejected: false,
+  });
+
+  const selectedNviStatuses = Object.entries(nviStatusFilter)
+    .filter(([, selected]) => selected)
+    .map(([status]) => status);
+
+  const nviStatusQuery =
+    selectedNviStatuses.length > 0
+      ? `(${selectedNviStatuses.map((status) => `approvals.approvalStatus:${status}`).join(' OR ')})`
+      : '';
+
   const nviCandidatesQuery = useQuery({
     enabled: isOnNviCandidatesPage,
-    queryKey: ['nviCandidates', rowsPerPage, apiPage],
-    queryFn: () => fetchNviCandidates(rowsPerPage, apiPage * rowsPerPage),
+    queryKey: ['nviCandidates', rowsPerPage, page, nviStatusQuery],
+    queryFn: () => fetchNviCandidates(rowsPerPage, (page - 1) * rowsPerPage, nviStatusQuery),
     meta: { errorMessage: t('feedback.error.get_nvi_candidates') },
   });
 
-  const typeBuckets = ticketsQuery.data?.aggregations?.type.buckets ?? [];
-  const doiRequestCount = typeBuckets.find((bucket) => bucket.key === 'DoiRequest')?.docCount;
-  const publishingRequestCount = typeBuckets.find((bucket) => bucket.key === 'PublishingRequest')?.docCount;
-  const generalSupportCaseCount = typeBuckets.find((bucket) => bucket.key === 'GeneralSupportCase')?.docCount;
-
-  const statusBuckets = ticketsQuery.data?.aggregations?.status.buckets ?? [];
-  const newCount = statusBuckets.find((bucket) => bucket.key === 'New')?.docCount;
-  const pendingCount = statusBuckets.find((bucket) => bucket.key === 'Pending')?.docCount;
-  const completedCount = statusBuckets.find((bucket) => bucket.key === 'Completed')?.docCount;
-  const closedCount = statusBuckets.find((bucket) => bucket.key === 'Closed')?.docCount;
+  const nviPendingCount = '';
+  const nviApprovedCount = '';
+  const nviRejectedCount = '';
```

Comment:
Får disse en verdi senere?

Reply:
Jepp, fra PR-description: "Aggregeringene skal også endres på, så har bare lagt inn støtte for det, uten å ta det i bruk enda."
Skulle lagt kommentaren på linjene det gjelder i stedet, så det hadde vært tydeligere